src/test/java/lotto/domain/LottoTest.java

```diff
@@ -17,29 +17,37 @@ public class LottoTest {
     @Test
     @DisplayName("생성자 의존성 주입 - 로또 번호")
     void constructorDI() {
-        assertThat(new Lotto(lottoNumbers()).lottoNumbers().size()).isEqualTo(45);
+        assertThat(new Lotto(LottoNumber.lottoNumbers().subList(0,6)).lottoNumbers().size()).isEqualTo(6);
     }
 
     @Test
     @DisplayName("생성자 의존성 주입 - 당첨 번호 길이 예외")
-    void constructorDIExceptionForWinningNumberLength() {
+    void constructorDIExceptionForNumberLength() {
         assertThatThrownBy(() -> new Lotto(LottoNumber.lottoNumbers().subList(0, 7))).isInstanceOf(IllegalArgumentException.class);
```

Comment:
```suggestion
        assertThatThrownBy(() -> new Lotto(LottoNumber.lottoNumbers().subList(0, 7)))
                .isInstanceOf(IllegalArgumentException.class);
```

1. 이렇게 줄바꿈이 한 번 들어가면 가독성이 더 좋아질 것 같아요.
2. `LottoNumber.lottoNumbers().subList(0, 7)`처럼 배경지식을 요하는 메서드를 사용하기보다, 딱 봐도 바로 어떤 숫자가 들어가는지 알 수 있게 하면 명확성, 가독성 등에서 더 좋지 않을까요? 숫자를 넣으면 바로 해당 숫자들로 `List<LottoNumber>`를 만드는 메서드를 테스트 코드에 만드는 방법은 어떨까요?